sql-odbc/src/sqlodbc/opensearch_communication.cpp

```diff
@@ -900,10 +902,12 @@ std::string OpenSearchCommunication::GetServerDistribution() {
 
     std::shared_ptr< Aws::Http::HttpResponse > response =
         IssueRequest("", Aws::Http::HttpMethod::HTTP_GET, "", "", "");
-    if (response == nullptr) {
+    if (response == nullptr || response->GetResponseCode() == Aws::Http::HttpResponseCode::REQUEST_NOT_MADE) {
```

Comment:
When does get `REQUEST_NOT_MADE`?

Reply:
When a URL cannot be reached by curl.

Reply:
Do you think this is what #280 was running into?

Reply:
Can't say entirely for certain. When testing I noticed this `if` statement was never entered in the event of failing to query the SQL plugin. A lot of the users found the same error message about the SQL plugin not being installed. It leads me to believe that the query for SQL plugin is the culprit, but with the diagnostics that were previously in the code I can't say for certain.